pkg/errors/errors.go

```diff
@@ -233,7 +233,27 @@ func NewUserNotAllowedError(image string, onbuild bool) error {
 	return Error{
 		Message:    msg,
 		ErrorCode:  UserNotAllowedError,
-		Suggestion: fmt.Sprintf("modify image %q to use a numeric user within the allowed range or build without the --allowed-uids flag", image),
+		Suggestion: fmt.Sprintf("modify image %q to use a numeric user within the allowed range, or build without the --allowed-uids flag", image),
```

Comment:
i think this error gets dumped inside of openshift also, so suggesting passing flags is going to be confusing in that context.

Reply:
there are other spots where we do this. Created #880 to address.